packages/financial-templates-lib/test/clients/FinancialContractClient.js

```diff
@@ -71,28 +76,40 @@ const versionedIt = function (supportedVersions, shouldBeItOnly = false) {
 
 const Convert = (decimals) => (number) => parseFixed(number.toString(), decimals).toString();
 
-contract("FinancialContractClient.js", function (accounts) {
-  const sponsor1 = accounts[0];
-  const sponsor2 = accounts[1];
+describe("FinancialContractClient.js", function () {
+  before(async function () {
+    accounts = await web3.eth.getAccounts();
+    [sponsor1, sponsor2] = accounts;
+  });
+
+  let sponsor1, sponsor2;
```

Comment:
just a nit but probably would be clearer to define this above, like you do in the next file

Reply:
Good point. Will do.